### PR TITLE
Release `v6.4.0`

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: runtime/crab/src/migration.rs [hunk 1]
```diff
@@ -41,7 +41,16 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 }
 
 fn migrate() -> frame_support::weights::Weight {
-	frame_support::weights::Weight::zero()
+	// substrate
+	use frame_support::traits::StorageVersion;
```
Comment (boundless-forest) [inline runtime/crab/src/migration.rs:45]: These changes are also needed for the darwinia runtime?

Comment (AurevoirXavier) [reply runtime/crab/src/migration.rs:45]: No

File: runtime/crab/src/migration.rs [hunk 2]
```diff
@@ -41,7 +41,16 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
...
+
+	// Some pallets ere added on chain after the migration.
+	// Thus, they never required the migration and we just missed to set the correct
+	// `StorageVersion`.
+	StorageVersion::new(4).put::<Scheduler>();
+	StorageVersion::new(1).put::<PolkadotXcm>();
```
Comment (jiguantong) [inline runtime/crab/src/migration.rs:51]: What's the meaning of this line? What would happen if we don't set it?

Comment (AurevoirXavier) [reply runtime/crab/src/migration.rs:51]: https://github.com/itering/internal-security-audit/pull/53

File: runtime/crab/src/migration.rs (hunk 1 above)
Comment (boundless-forest) [reply runtime/crab/src/migration.rs:45]:
```sh
2023-08-30 13:39:11.238  INFO main runtime::frame-support: ⚠️ XcmpQueue declares internal migrations (which *might* execute). On-chain `StorageVersion(3)` vs current storage version `StorageVersion(2)`
2023-08-30 13:39:11.238  INFO main runtime::frame-support: ⚠️ DmpQueue declares internal migrations (which *might* execute). On-chain `StorageVersion(2)` vs current storage version `StorageVersion(1)`
```
I analysed the darwinia try-runtime result and found that these two pallet also break the try-runtime process. If we do nothing at all, the darwinia runtime try-runtime still panic if I understand correctly.

Comment (AurevoirXavier) [reply runtime/crab/src/migration.rs:45]: Please check the audit report.

**Review by boundless-forest — APPROVED** [review]

**Review by jiguantong — DISMISSED** [review]

**Pull request merged**